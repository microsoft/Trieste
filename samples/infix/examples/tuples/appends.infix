print "empty and empty" append((,), (,));
print "empty append" append(,);
print "unary append" append((,),);
print "nested appends" append((1, 2), append((3,), (4,)));