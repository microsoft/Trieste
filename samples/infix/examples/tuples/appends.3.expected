//! --enable-tuples --tuples-require-parens postfix
//! --enable-tuples --tuples-require-parens --use-parser-tuples postfix
//! --enable-tuples postfix
"empty and empty" 0 tuple 0 tuple 2 append print
"empty append" 0 append print
"unary append" 0 tuple 1 append print
"nested appends" 1 2 2 tuple 3 1 tuple 4 1 tuple 2 append 2 append print
